backend/src/schema/resolvers/posts.js

```diff
@@ -80,24 +80,27 @@ export default {
       params = await fileUpload(params, { file: 'imageUpload', url: 'image' })
       params.id = params.id || uuid()
       let post
-
```

Comment:
> [<img alt="appinteractive" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/appinteractive) **Authored by [appinteractive](https://github.com/appinteractive)**
_<time datetime="2019-09-11T15:00:52Z" title="Wednesday, September 11th 2019, 5:00:52 pm +02:00">Sep 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -79,6 +79,7 @@ export default {
       delete params.categoryIds
       params = await fileUpload(params, { file: 'imageUpload', url: 'image' })
       params.id = params.id || uuid()
+      params.createdAt = params.createdAt || new Date().toISOString()
```


We should really always working with adding utc dates to avoid issues down the road. 

Also: why the heck is it possible to overwrite the createdAt date, this should never, ever happen.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-11T16:57:34Z" title="Wednesday, September 11th 2019, 6:57:34 pm +02:00">Sep 11, 2019</time>_

---


good question... I pushed up some code that would set the `created_at` in a cypher statement, and then parse the `post.created_at` to return a String as `createdAt`... it's not done yet, but I am for the day... 
would you and/or @roschaefer have a look and tell me if you don't like it/want to go in a different direction?